Sources/MapboxMaps/Location/Puck/Puck2D.swift

```diff
@@ -238,6 +251,40 @@ internal final class Puck2D: Puck {
     }
 }
 
+extension Puck2D: DisplayLinkParticipant {
+    func participate() {
+        guard isActive,
```

Comment:
this [line](https://github.com/mapbox/mapbox-maps-ios/pull/1513/files#diff-49c72aa33067d00c9bc906b7e1e0d4e6d56bd77db95450efcf80115bc157307fR23) will register `DisplayLinkParticipant` if the puck is active and pulse is enabled, can we safely assume that this function will get executed only if the puck is active and remove this check here?

Reply:
Yeah, this check is removed!